pkg/core/native/management.go

```diff
@@ -525,16 +525,14 @@ func (m *Management) PutContractState(d dao.DAO, cs *state.Contract) error {
 }
 
 func (m *Management) getNextContractID(d dao.DAO) (int32, error) {
-	var id = big.NewInt(1)
 	si := d.GetStorageItem(m.ContractID, keyNextAvailableID)
-	if si != nil {
-		id = bigint.FromBytes(si.Value)
-	} else {
-		si = new(state.StorageItem)
-		si.Value = make([]byte, 0, 2)
+	if si == nil {
+		return 0, errors.New("nextAvailableID is not initialized")
+
 	}
+	id := bigint.FromBytes(si.Value)
```

Comment:
Why don't we use `getInt...` here?

Reply:
It's a bit different situation, because we need to put value back into DB right after getting it. We use `toPreallocatedBytes` here, so storage item is necessary (not only its value).

So I think it's a kind of optimisation.